clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoMetalake.java

```diff
@@ -35,7 +36,7 @@
  * as sub-level metadata collections. With {@link GravitinoMetalake}, users can list, create, load,
  * alter and drop a catalog with specified identifier.
  */
-public class GravitinoMetalake extends MetalakeDTO implements SupportsCatalogs {
+public class GravitinoMetalake extends MetalakeDTO implements SupportsCatalogs, SupportsCatalog {
```

Comment:
Do we still need this GravitinoMetalake? I think maybe it can be hidden behind the client?

Reply:
For the GravitinoAdminClient, which only load/create/alter/drop Metalakes, then for the loaded metalake objects, the user may further list/create/drop catalogs. So put these operations in the Metalake object level is to facilitate those operations.

Besides, the list/create/drop catalog operations on GravitinoClient object are delegated to the default metalake object. So, I think keep these methods at the "GravitinoMetalake" class is okay, will benefit both clients.